packages/components/src/ConnectionCard/TripSector.js

```diff
@@ -0,0 +1,152 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import {
+  Text,
+  StyleSheet,
+  type StylePropType,
+  CarrierLogo,
+  Icon,
+} from '@kiwicom/universal-components';
+
+type Props = {|
+  +arrival: string,
+  +arrivalTime: string,
+  +carrier: $ElementType<
+    $PropertyType<React.ElementProps<typeof CarrierLogo>, 'carriers'>,
+    number,
+  >,
+  +tripDate: string,
+  +departure: string,
+  +departureTime: string,
+  +duration: string,
+|};
+
+const TimelineArrow = () => (
+  <View style={styles.timelineArrowContainer}>
+    <View style={styles.circle} />
+    <View style={[styles.line, styles.firstLine]} />
+    <Icon
+      name="chevron-down"
+      color={defaultTokens.paletteInkLighter}
+      size="small"
+      style={styles.timelineArrowIcon}
+    />
+    <View style={[styles.line, styles.secondLine]} />
+    <View style={styles.circle} />
+  </View>
+);
+
+export default class TripSector extends React.Component<Props> {
+  renderText = (text: string, style: StylePropType) => (
+    <Text style={[styles.text, style]} numberOfLines={1}>
+      {text}
+    </Text>
+  );
+
+  render() {
+    const {
+      arrival,
+      arrivalTime,
+      carrier,
+      departure,
+      departureTime,
+      duration,
+      tripDate,
+    } = this.props;
+    return (
+      <View style={styles.container}>
+        <View style={styles.row}>
+          <View style={styles.carrierLogo}>
+            {/* <CarrierLogo size="medium" carriers={[carrier]} /> */}
+            <Text>CarrierLogo@TODO</Text>
+          </View>
+          <View style={styles.tripItems}>
+            <View style={styles.time}>
+              <Text numberOfLines={1}>{departureTime}</Text>
+              <Text numberOfLines={1}>{arrivalTime}</Text>
+            </View>
+            <TimelineArrow />
+            <View style={styles.places}>
+              {this.renderText(departure)}
+              {this.renderText(arrival)}
```

Comment:
This does nothing a component does not

It is better to create a component. That is what react is for 🙂